From: Marc Kleine-Budde <mkl@pengutronix.de>
Subject: disable ppc memcpy that needs kernel kernel config file

ppcasm_memcpy_cachable.S needs the kernel config to compile.
This patch simply disables the file.

Signed-off-by: Marc Kleine-Budde <mkl@pengutronix.de>

---
 lib/direct/Makefile.am |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: DirectFB-1.3.0/lib/direct/Makefile.am
===================================================================
--- DirectFB-1.3.0.orig/lib/direct/Makefile.am
+++ DirectFB-1.3.0/lib/direct/Makefile.am
@@ -21,7 +21,7 @@ EXTRA_DIST = \
 
 if BUILDPPCASM
 if HAVE_LINUX
-ppcasm_sources = ppcasm_memcpy.S ppcasm_memcpy_cachable.S
+ppcasm_sources = ppcasm_memcpy.S #ppcasm_memcpy_cachable.S
 else
 ppcasm_sources = ppcasm_memcpy.S
 endif
